app/utils.py

```diff
@@ -41,18 +41,11 @@ def __init__(self, bot_path, item_type):
         g = Github(access_token)
         self.repo = g.get_repo(repo_name)
 
-        self.existing_files = set(
-            pr_file.filename
-            for pr in self.repo.get_pulls(state="open")
-            for pr_file in pr.get_files()
-            if pr_file.filename.startswith(self.bot_path)
-        )
-        git_tree = self.repo.get_git_tree(self.repo.default_branch, recursive=True)
-        self.existing_files.update(
-            file.path
-            for file in git_tree.tree
-            if file.path.startswith(self.bot_path) and file.path.endswith(".md")
-        )
+        self.existing_files = set()
+        for pr in self.repo.get_pulls(state=["open", "closed"]):
```

Comment:
you can filter here more: https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests

e.g. for head or maybe base?

Reply:
https://github.com/usegalaxy-eu/galaxy-social-assistent/pull/3/commits/98266ec86953d89384ea80f6d91e83301e773f83